docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,374 @@
+# JupyterLab
+
+ - Even if you don't use Jupyter, you can still take this survey. Just indicate this fact in the first question 
+   and carry on as best as you can.
+ - Thank you. Your participation guides Jupyter's roadmap toward your real-life use cases by quantifiably 
+   helping us prioritize the functionality that is important to our userbase. 
+ - So that you know what to expect, it's comprised of 20 questions spread across the sections below. 
+   As a fair heads up, Question 7  is the biggest one, but it provides critical information.
+    - Usage patterns.
+    - Data
+    - Visualization.
+    - Scale.
+    - Collaboration.
+ - The aggregate survey data itself will be openly shared with the Jupyter community when polling closes in mid-December.
+   If you opt to provide your email address for a user interview, it not be used for Jupyter's promotional purposes and 
```

Comment:
Missing word.
```suggestion
   If you opt to provide your email address for a user interview, it will not be used for Jupyter's promotional purposes and 
```

Reply:
Thank you. Added 'will'

Reply:
@LayneSadler Did you push these changes for missing words? I don't see them on the PR

Reply:
@marthacryan No, I did not.